lib/widgets/screens/explore/recommended_mentors_filters.dart

```diff
@@ -140,16 +155,16 @@ class _ExpertiseState extends State<_Expertise> {
               ? theme.colorScheme.onInverseSurface
               : theme.colorScheme.surface),
         ),
-        child: Text(l10n.exploreSearchFilterSkills(skill),
+        child: Text(widget.skillTranslate(skill) ?? skill,
             style: TextStyle(color: theme.colorScheme.primary)),
```

Comment:
This doesn't seem to be working. Not sure what the cause is, but I can see that the UI is displaying the `textId` rather than the `translatedValue`:

<img width="513" alt="Screenshot 2023-09-28 at 12 47 04 AM" src="https://github.com/micromentor-team/mm-flutter-app/assets/12814577/6f98e6b6-e726-4e05-9cee-1feb3371e56f">
<img width="509" alt="Screenshot 2023-09-28 at 12 47 09 AM" src="https://github.com/micromentor-team/mm-flutter-app/assets/12814577/7e417b7c-caa2-435c-9572-3c5df8ea1ca2">
<img width="529" alt="Screenshot 2023-09-28 at 12 47 18 AM" src="https://github.com/micromentor-team/mm-flutter-app/assets/12814577/977c4770-a8cc-4328-a100-98ec8f5838f1">

Reply:
I'm assuming this was not possible to test due to the issue with the remote server `Language`s mentioned in another comment

Reply:
Ouch. Very true. I was able to test that but for some reason it worked back then. At the state of reviewed PR when an item is selected it'd be expanded into full, translated string. But for some reason this doesn't work for autoselect items.

Reply:
Fixed by adding: `displayStringForOption: optionsTranslations` in `Autocomplete`.